package.json

```diff
@@ -107,5 +109,18 @@
     "socketcluster": "4.1.0",
     "socketcluster-client": "4.1.2",
     "url": "0.11.0"
+  },
+  "xo": {
+    "esnext": true,
+    "space": 2,
+    "env": [
+      "browser",
+      "node",
+      "webpack"
+    ],
+    "extends": "xo-react",
+    "globals": [
+      "__PRODUCTION__"
+    ]
   }
 }
```

Comment:
I'm not a big fan of xo because of this specific aspect of it... polluting the package.json
What are your thoughts on eslint?